dev-libs/pigpio/pigpio-73.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_{6,7,8}} )
```

Comment:
Let's not, though ;)

The Python team has asked us try to get rid of 2.7 while bumping. This package doesn't have any revdeps, so it's amongst the easiest to drop python2.7 already. 

You have 6 version with ~ keywords. Would it be time to clean some old versions from the tree as well? (as a separate commit though)